src/ui/controls/edit.cpp

```diff
@@ -1256,14 +1260,25 @@ void CEdit::DrawColor(Math::Point pos, Math::Point dim, Gfx::Color color)
 // Give the text to edit.
 
 void CEdit::SetText(const char *text, bool bNew)
-{
+{
     int     i, j, font;
     bool    bBOL;
+
+    if ( text == nullptr )

```

Comment:
For handling of wrong function arguments it's a lot better to use `assert()` than to fail silently.